### ISSUE TITLE
Verify existence of RASCAL.MF when starting dev servers

### ISSUE THREAD

**Issue by rodinaarssen** [usethesource/rascal-language-servers#108]
If this is missing in `rascal-lsp/target/classes`, `rascal-lsp` is not available

**Comment by DavyLandman** [comment]
Is this during second-level debugging? Or which mode are you encountering this?

**Comment by rodinaarssen** [comment]
Yes, indeed. It's supposed to be there after `mvn package`, but I suppose IDEs may cause recompilation and hence removal

**Comment by DavyLandman** [comment]
I think we can be safe to say that rascal-lsp should always be added for the parametric evaluator. So maybe we'll hardcode it? 

Or is it the case that's it's already hardcoded, just not picked up?

**Comment by jurgenvinju** [comment]
Yes. It's the case when Rascal.MF is not yet copied to the target folder

**Comment by rodinaarssen** [comment]
Indeed. It is hardcoded as a dependency, but if the actual `RASCAL.MF` file is missing, it will not appear on the search path

**Comment by DavyLandman** [comment]
Okay, sorry for being slow on the uptake, but it happens when someone is running viscose extension in second level, while also developing a DSL?

**Comment by rodinaarssen** [comment]
When registering a language in a second level terminal (against the dev servers)

**Comment by DavyLandman** [comment]
We're still getting hit by this. The maven emulation of the nested eclipse instance in the java extension cleans out the target folder if the `pom.xml` is changed, but doesn't properly run/emulate the `process-resources` that would copy back the `RASCAL.MF` in the target folder.

**Comment by DavyLandman** [comment]
Looks like https://github.com/eclipse-m2e/m2e-core/issues/1511 is related, so one cause of the issue should be solved by now, but we're still seeing it.

It seems to be caused by the fact that the resource definition points to the root folder
